docs/datatypes/mesh/index.md

```diff
@@ -31,3 +32,36 @@ The following pictures depict a 3D mesh and a point cloud with 1000 samples from
 ```{figure} pointcloud-man.gif
 :width: 50%
 ```
+
+## Mesh
```

Comment:
the section title is `3D Mesh` above, and then we explain point cloud and go back to mesh.
It feels a bit weird to me, maybe we should name this one `Vertices and Faces representation` and the other one `point cloud representation` ?

Reply:
yes I agree